### PR TITLE
[#31043] Add top-footer into style guide.

### PR DESCRIPTION
**Pull request by HierenFeb**
Style background for Top Footer region and list link in it.

### CODE REVIEW

**Comment by kimpepper** [comment]
test this please

File: themes/agov/agov_whitlam/sass/components/top-footer/top-footer.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "content": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
+}
```
Comment (kimpepper) [inline themes/agov/agov_whitlam/sass/components/top-footer/top-footer.json]: Always leave a single new line at the end of a file. See https://github.com/editorconfig/editorconfig/wiki/Newline-at-End-of-File-Support for editor support.

**Comment by JohnAlbin** [comment]
All the comments on https://github.com/previousnext/agov/pull/507 apply to this PR as well.

You'll need to add a new footer_top region to the theme and add that to page.tpl.php.

**Comment by kimpepper** [comment]
Feedback has been resolved.

**Pull request merged**